.github/workflows/macos-installer.yml

```diff
@@ -44,11 +44,7 @@ jobs:
           path: "./out/macos-universal/crc-macos-installer.pkg"
       - name: Install crc pkg
         run: sudo installer -pkg out/macos-universal/crc-macos-installer.pkg -target /
-      - name: Set podman preset as part of config
-        run: crc config set preset podman
+      - name: Set microshift preset as part of config
+        run: crc config set preset microshift
       - name: Run crc setup command
-        run: crc setup
-      - name: Wait 10 sec for the daemon to serve
-        run: sleep 10
-      - name: Run crc with podman preset
-        run: crc start
```

Comment:
The commit log is
```
 modified github workflow

removing crc start from github test
```

This describes what the commit is doing, but this does not explain the reason why you want to make this change. My guess is that this is required as we cannot reliably run `crc start` for the microshift preset in github actions because the runners are not powerful enough, but this is just a guess, maybe you have a very different reason for this change, and I cannot know that from just the commit log and the diff. Can you explain this in the git log?

Reply:
The main reason is that with podman preset, authentication was not required from user's end.  As per previous discussions, `crc start` with `microshift` preset is not necessary as a part of the github workflow. @praveenkumar is that correct?

Reply:
I'm still not clear what "authentication" refers to here? Does this refer to the pull secret which is needed for `crc start` with microshift? And since we don't have a valid pull secret in gh actions, we can't run crc start?

Reply:
Yes, that's what I mean. The pull secret requirement means new steps have to be added in the github actions. This step was added in [this commit.](https://github.com/vyasgun/crc/commit/a4c61d9e9c2797aa6487c8f219bef26c5d045d58) I had a discussion with @praveenkumar about this and it seemed like a better idea to remove the `start` command.